shell/platform/embedder/tests/embedder_unittests_proctable.cc

```diff
@@ -0,0 +1,60 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/embedder/embedder.h"
+
+#include "flutter/testing/testing.h"
+
+namespace flutter {
+namespace testing {
+
+// Verifies that the proc table is fully populated.
+TEST(EmbedderProcTable, AllPointersProvided) {
+  FlutterEngineProcTable procs = {};
+  procs.struct_size = sizeof(FlutterEngineProcTable);
+  FlutterEngineGetProcAddresses(&procs);
```

Comment:
Here and elsewhere, please assert the return value of `FlutterEngineGetProcAddresses`.

Reply:
Done.